config/global.json

```diff
@@ -3232,5 +3235,125 @@
             ],
             "comment": "In recent years, deep learning models have considerably advanced the performance of segmentation tasks on Brain Magnetic Resonance Imaging (MRI). However, these models show a considerable performance drop when they are evaluated on unseen data from a different distribution. Since annotation is often a hard and costly task requiring expert supervision, it is necessary to develop ways in which existing models can be adapted to the unseen domains without any additional labelled information. In this work, we explore one such technique which extends the CycleGAN [2] architecture to generate label-preserving data in the target domain. The synthetic target domain data is used to train the nn-UNet [3] framework for the task of multi-label segmentation. The experiments are conducted and evaluated on the dataset [1] provided in the ‘Cross-Modality Domain Adaptation for Medical Image Segmentation’ challenge [23] for segmentation of vestibular schwannoma (VS) tumour and cochlea on contrast enhanced (ceT1) and high resolution (hrT2) MRI scans. In the proposed approach, our model obtains dice scores (DSC) 0.73 and 0.49 for tumour and cochlea respectively on the validation set of the dataset. This indicates the applicability of the proposed technique to real-world problems where data may be obtained by different acquisition protocols as in [1] where hrT2 images are more reliable, safer, and lower-cost alternative to ceT1."
         }
+    },
+    "00022_WGAN_CARDIAC_AGING": {
+        "execution": {
+            "package_name": "00022_WGAN_CARDIAC_AGING",
+            "package_link": "https://zenodo.org/record/7446930",
+            "model_name": "model",
+            "extension": ".ckpt",
+            "image_size": [
+                256,
+                256
+            ],
+            "dependencies": [
+                "nibabel==3.2.1",
+                "pytorch-lightning==1.4.7",
+                "pandas",
+                "comet-ml",
+                "monai",
+                "grad-cam",
+                "matplotlib",
+                "monai[skimage]",
+                "munch==2.5.0",
+                "pillow==7.0.0",
```

Comment:
Maybe try `pillow>=7.0.0`

Reply:
I will try that, but this doesn't seem to be the future proof solution. The more models we integrate the harder it will be to have a one set of dependencies that work for every model. 
As a solution, probably we will have to perform testing independently for each model (setup environment every time) this way we will also solve any memory problems due to loading all the models at once.

Reply:
This might help us: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs
We could run install_model_dependencies.py for each particular model in a separate job. So there will be no requirement conflicts. Also we will see much more clearly which models are passing tests and which are not.

Reply:
That's a great idea!! 👍